Add files via upload (#50)
SEKOIA-IO/documentation: docs/integrations/rsyslog_generic_settings.md

- Restart the rsyslog service: 
```bash
sudo systemctl restart rsyslog.service
```

## Troubleshoot
After setting up your rsyslog, you may face issues due to contextyal environment or error during copy pasting.

### Rsyslog daemon
Ensure the rsyslog service is currrently running on the server
```bash
ps -A | grep rsyslog 
```
You should see a line with rsyslogd daemon. If not, try to restart the service:
```bash
sudo systemctl restart rsyslog.service
```

Also insure the following lines are not commented in the configuration file `/etc/rsyslog.conf`:
```bash
# provide TCP syslog reception
module(load="imtcp")
input(type="imtcp" port="514")

# provides UDP syslog reception
module(load="imudp")
input(type="imudp" port="514")
```

### Local messages
Ensure the logs are received on the Rsyslog, meaning:
- Configurations are correctly undertaken on the remote equipements
- Internal network flows are open on `TCP 514`

To check it, run the following command:
```bash
tail -n 15 /var/log/syslog
``` 

### Forwarded messages to SEKOIA.IO
Ensure the connexion is `ESTABLISHED` between the rsyslog server and SEKOIA.IO. To do so, please run the following command:
```bash

> sudo apt install net-tools -f -y && sudo netstat -taupn | grep syslog 

Reviewer: ```suggestion
sudo ss -ltp | grep syslog 
```

```

> If the connexion is not establish, and the previous status are operational, it is possible that a file `xx-<technology>.conf` has a typography.

Reviewer: ```suggestion
If the connexion is not established, and the previous status are operational, it is possible that a file `xx-<technology>.conf` has a typography.
```

By experience, most of the time there is a misconfiguration in the `IF` condition.

> For example, if you receive the fortigate logs in `/var/log/syslog` but not in the plateforme, you can follow those steps:

Reviewer: ```suggestion
For example, if you receive the Fortigate logs in `/var/log/syslog` but not in SEKOIA.IO, you can follow those steps:
```

- First try to log the raw events to a local file, by appending the following line in the configuration file:
```bash
*.* then /tmp/fortigate.log;SEKOIAIOFortigateTemplate
```

> - Restart the rsyslog service

Reviewer: ```suggestion
- Restart the Rsyslog service
```
